src/pykeen/models/base.py

```diff
@@ -379,6 +379,9 @@ def predict_r(
         """
         self.eval()  # Enforce evaluation mode
         ht_batch = ht_batch.to(self.device)
+        # Checking for self.inverse is not necessary since we have score_h_inverse.
+        # Since we have inverse triples, where the order of entities is inverted,
+        # and the relation replaced by an inverse relation.
```

Comment:
should `predict_r` compute scores for inverse relations? I think: no

Reply:
this is related to: https://github.com/pykeen/pykeen/pull/726

Reply:
Can we reinvestigate this now that #726 is done?

Reply:
I think, it should only score "real" relations. maybe #752 brings more clarity.